### ISSUE TITLE
Ebook conversion

### ISSUE THREAD

**Issue by MrJosu** [C4illin/ConvertX#191]
Can this option be added or it's there and I am missing something?
I am sorry If i put this in wrong place

**Comment by C4illin** [comment]
Pandoc can do some formats, otherwise de need to add Calibre

**Comment by MrJosu** [comment]
> Pandoc can do some formats, otherwise we need to add Calibre

will be nice to have full features to be honest^^

**Comment by C4illin** [comment]
> > Pandoc can do some formats, otherwise we need to add Calibre
> 
> will be nice to have full features to be honest^^

What features are you missing?

**Comment by kfoxton** [comment]
mobi to epub?

**Comment by C4illin** [comment]
should work in the `:main` image now, try it out!

**Comment by C4illin** [comment]
currently it makes the docker image 50% larger, around 200 MB since it needs to install the entire calibre package

**Comment by Aymendje** [comment]
Yeah, I looked into Calibre for some other projects, and their is no way to only get the ebook-converter, you need the full suite
One thing we could add is maybe an ENV for "ebooks" and if detected, will download calibre automatically and add it ?

**Comment by C4illin** [comment]
I tried to install all deps seperately and copy over the calibre install files but it doesn't work.

I think an ENV is lots of extra work for a small benefit. An alternative would be this project https://github.com/gryf/ebook-converter but it seems a bit abandoned

**Comment by Aymendje** [comment]
Other projects, such as [Stirling-PDF](https://github.com/Stirling-Tools/Stirling-PDF/pkgs/container/stirling-pdf) have different packages, for example `stirling-ultra-lite` or `sterling-fat`, etc.
You could have a similar approach, but I think managing multiple build just for calibre isn't worth it

Extracting the `ebook-converter` executable from calibre wont work, since its actually just a python file that loads into calibre-cli

**Comment by C4illin** [comment]
I think having multiple images makes more sense in this case. But I think most people would want calibre anyways even if the install size is 200 MB?
